Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -181,6 +179,10 @@ private void OnReclaimCancelled(ReclaimCancelledEvent args)
         }
         private void StartProcessing(EntityUid toProcess, BiomassReclaimerComponent component)
         {
+            PhysicsComponent? physics = null;
```

Comment:
just place this in the function parameters (most of SS14 does that if the function isn't a registered event listener)